Source/Core/DiscIO/VolumeCreator.cpp

```diff
@@ -211,16 +209,13 @@ static IVolume* CreateVolumeFromCryptedWiiImage(IBlobReader& _rReader, u32 _Part
 EDiscType GetDiscType(IBlobReader& _rReader)
 {
 	CBlobBigEndianReader Reader(_rReader);
-	u32 WiiMagic = Reader.Read32(0x18);
-	u32 WiiContainerMagic = Reader.Read32(0x60);
+	u32 WiiMagic = Reader.Read32(0x18); // Disc layout magic (Wii)
 	u32 WADMagic = Reader.Read32(0x02);
-	u32 GCMagic = Reader.Read32(0x1C);
+	u32 GCMagic = Reader.Read32(0x1C); // Disc layout magic (GC)
 
 	// check for Wii
-	if (WiiMagic == 0x5D1C9EA3 && WiiContainerMagic != 0)
-		return DISC_TYPE_WII;
-	if (WiiMagic == 0x5D1C9EA3 && WiiContainerMagic == 0)
-		return DISC_TYPE_WII_CONTAINER;
+	if (WiiMagic == 0x5D1C9EA3)
+		return DISC_TYPE_WII; //Previously named DISC_TYPE_WII_CONTAINER
```

Comment:
I don't think this comment should be here. Comments should contain things that are useful to know about the current code, not notes about obscure deleted things. There's always the git history for those who are interested.